tensorflow/contrib/layers/python/layers/layers.py

```diff
@@ -379,9 +379,11 @@ def convolution2d(inputs,
       If greater than 1, than the a'trous convolution is applied and `stride`
       must be set to 1.
     activation_fn: activation function.
+      set to None for linear activation
```

Comment:
rewrite as
activation_fn: activation function, set to None to skip it and maintain a linear activation.